src/setup/bootstrap/createStashRecords.ts

```diff
@@ -16,7 +16,18 @@ export const createSampleStashRecords = async ({
 }: CreateSampleStashRecordsInput) => {
   const stashClient = buildStashClient();
 
+  const sftpBucketName = requiredEnvVar("SFTP_BUCKET_NAME");
+  const outboundBucketPath = "trading_partners/ANOTHERMERCH/outbound";
+
   const partnership: Partnership = {
+    ack: {
+      enabled: true,
+      destination: {
```

Comment:
Did you consider putting a `997` entry in the `transactionSets` array? That would keep any "destinations" config in the same place, and allow multiple for 997.

Reply:
I'm not sure how that would work, actually. The `997`s don't require the use of a guide, so resolving the config / identifying the transaction set wouldn't work with out current mechanisms. What were you envisioning it to look like as an entry in `transactionSets`?

Also not sure what you meant by multiple 997 -- did you mean different content for different partnerships? (again with no guide and the use of `x12-tools` to generate the `997` content this doesn't seem necessary IMO)

Reply:
I didn't have a design in mind, but maybe "guideId" would become optional - we need that to allow base guides anyway?

By "allow multiple for 997" - I meant destinations - like RT today send 997s to the buckets (the default you've covered) but also send 997 to their data warehouse (stitch).

Reply:
Ok, refactored to align with the partner profiles data model (and to handle 997 acks as another type of transaction set). PTAL when you have a chance, @BDQ ! Thanks again for the feedback and for kicking ideas around on how this should look